.gitattributes

```diff
@@ -1,6 +1,6 @@
 # Convert all line endings for files detected as text to lf to avoid OS specific
 # line endings showing up in git diffs
-*                                             text=auto
+*                                             text=auto eol=lf
```

Comment:
Just adding a note, changing that might require windows users to reclone the repo so the existing file will not cause issue as shown in point 4 here: https://prettier.io/docs/en/options.html#end-of-line